lib/src/ui/pages/home/home_page.dart

```diff
@@ -1,67 +1,91 @@
 import 'package:allthenews/generated/l10n.dart';
 import 'package:allthenews/src/di/injector.dart';
-import 'package:allthenews/src/ui/common/util/dimens.dart';
 import 'package:allthenews/src/ui/common/util/untranslatable_strings.dart';
-import 'package:allthenews/src/ui/common/widget/primary_icon_button.dart';
+import 'package:allthenews/src/ui/common/widget/ny_times_appbar.dart';
 import 'package:allthenews/src/ui/pages/dashboard/dashboard_page.dart';
+import 'package:allthenews/src/ui/pages/home/home_page_notifier.dart';
 import 'package:allthenews/src/ui/pages/home/home_tab.dart';
 import 'package:allthenews/src/ui/pages/profile/profile_page.dart';
-import 'package:allthenews/src/ui/pages/settings/settings_notifier.dart';
-import 'package:allthenews/src/ui/pages/settings/settings_page.dart';
 import 'package:flutter/material.dart';
 import 'package:provider/provider.dart';
 
-abstract class _Constants {
-  static const appBarActionsVerticalPadding = 11.0;
-  static const appBarActionsIconsPadding = 8.0;
-  static const appBarTitleFontFamily = 'Chomsky';
-  static const appBarTitleLeftPadding = 10.0;
-}
-
 class HomePage extends StatefulWidget {
+  final int initialPage;
```

Comment:
w sumie HomePage nie musi trzymać initialPage, może to robić notifier co nie? 🤔

Reply:
to w ogóle można wyjebać